open_ce/images/builder/Dockerfile-p10

```diff
@@ -11,7 +11,7 @@ ENV BUILD_USER=builder
 ARG BUILD_ID=1084
 
 RUN export ARCH="$(uname -m)" && \
-    yum repolist && yum install -y rsync openssh-clients diffutils procps gcc-toolset-10 gcc-toolset-11 && \
+    yum repolist && yum install -y rsync openssh-clients diffutils procps git-lfs gcc-toolset-10 gcc-toolset-11 && \
```

Comment:
why do we still need gcc-toolset-10 ?

Reply:
The GCC11 change is in another PR @alexlang74. This PR is just installing git-lfs.

Reply:
Ok, thanks!